src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.graphql

```diff
@@ -13,25 +13,6 @@ query ContactDetailsTab($accountListId: ID!, $contactId: ID!) {
     envelopeGreeting
     preferredContactMethod
     timezone
-    people {
```

Comment:
So I noticed we had a fragment that was already grabbing this, so I moved my new stuff to that fragment and removed this.